common/app/views/support/Profile.scala

```diff
@@ -135,19 +135,30 @@ object ImgSrc extends Logging {
   }
 
   def srcset(imageContainer: ImageContainer, widths: WidthsByBreakpoint): String = {
-    widths.profiles.map { profile =>
-      if(ImageServerSwitch.isSwitchedOn) {
-        s"${findLargestSrc(imageContainer, profile).get} ${profile.width.get}w"
-      } else {
-        s"${findNearestSrc(imageContainer, profile).get} ${profile.width.get}w"
-      }
-    } mkString ", "
+    widths.profiles.map { profile => srcsetForProfile(profile, imageContainer) } mkString ", "
   }
 
-  def srcset(path: String, widths: WidthsByBreakpoint): String = {
-    widths.profiles map { profile =>
-      s"${ImgSrc(path, profile)} ${profile.width.get}w"
-    } mkString ", "
+  def srcsetForBreakpoint(breakpointWidth: BreakpointWidth, widths: WidthsByBreakpoint, maybeUrl: Option[String] = None, maybeImageContainer: Option[ImageContainer] = None) = {
+    widths.breakpointWidthToPixels(breakpointWidth)
```

Comment:
I wonder if the `breakpointWidthToPixels` method makes more sense on the `BreakpointWidth` method, but it needs scope of the `WidthsByBreakpoint`.